README.md

```diff
@@ -210,7 +210,9 @@ In addition, the below configurations exist.
 - **condition**: The condition of tasks to wait. Available values are `"all"` or `"any"`. (string, default: `"all"`)
 - **status**: The status of tasks to wait. Available values are `"PENDING"`, `"RUNNING"`, or `"STOPPED"` (string, default: `"STOPPED"`)
 - **ignore_failure**: Ignore even if any tasks exit with any status. This option is true, then the behaviour includes one of when **ignore_exit_code** is `true`. (boolean, default: `false`)
-- **ignore_exit_code**: Ignore even if any tasks exit with any exit code. When the containers of the task include one that does not have exit code, it is not ignored even if this option is `true`. (boolean, default: `false`) 
+- **ignore_exit_code**: Ignore even if any tasks exit with any exit code. When the containers of the task include one that does not have exit code, it is not ignored even if this option is `true`. (boolean, default: `false`)
+- **delay_strategy**: The delay strategy of tasks to wait. Available values are `"fixed"` or `"exponential_backoff"`. (string, default: `"fixed"`)
```

Comment:
Could you explain your exponential backoff logic? Exponential backoff has various kinds of implementations.

And if you do not mind, I want to follow the exponential backoff logic: [digdag _retry](https://github.com/treasure-data/digdag/blob/0fb61bb/digdag-plugin-utils/src/main/java/io/digdag/util/RetryControl.java) because I do not want to make implementations of various types of exponential backoff in the digdag ecosystem.

Reply:
Then, I suggest the below config style and supporting the same option as [digdag _retry](https://github.com/treasure-data/digdag/blob/0fb61bb/digdag-plugin-utils/src/main/java/io/digdag/util/RetryControl.java) .

```yaml
+wait:
  ecs_task.wait>:
  cluster: my-cluster
  ...
  polling_strategy:
    interval_type: exponential
    limit: 3
    interval: 10
```

Reply:
I implemented exponential backoff with jitter.
It has little randomness when retry.
I referenced aws api retry document(https://docs.aws.amazon.com/ja_jp/general/latest/gr/api-retries.html), says most exponential backoff are with jitter. 
So I implemented.

But I think back, in this case, request conflicts may be rarely happens.
And digdag _retry exponential backoff implemented without jitter, I follow it.(thanks for your suggestion & instruction! I follow them. )

Reply:
I fixed exponential backoff strategy. thanks!
https://github.com/civitaspo/digdag-operator-ecs_task/pull/43/commits/e21a1a237e6b837eb6101cfaa5a02e64b556847b#diff-5c8445c113918dca16afe00a569c3f1e